### PR TITLE
added django registration templates and ajdust the styling

### LINKED ISSUE #83: Kria django rejistrasaun

### PR DESCRIPTION
**Pull request by marobo**
Closes #83 

## Description
added django registration templates and ajdust the styling

## For testing
- add this email backend `EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'` to your "local_settings.py" for supporting reset your password  during development
- Try login, reset password and change password see what's happening. :+1:

### CODE REVIEW

**Comment by marobo** [comment]
@MarioAP @joanico @eraulo , hau remata tiha ona PR nee, favor bele review.. Thanks

File: locale/tet/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -40,69 +54,284 @@ msgstr "Tama"
...
+"Please enter your old password, for security's sake, and then enter your new"
+" password twice so we can verify you typed it in correctly."
+msgstr ""
+"Favor ida hatama ita nia password tuan, ba questaun seguransa, no depois "
+"hatama ita nia password foun dala rua para ami bele verifika se ita hakerek "
+"ne'e loos."
+
+#: map/templates/registration/password_change_form.html:42
+#| msgid "Change password"
+msgid "Change my password"
```
Comment (eraulo) [inline locale/tet/LC_MESSAGES/django.po:218]:
```suggestion
msgid "Change your password"
```

File: locale/tet/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -40,69 +54,284 @@ msgstr "Tama"
...
 
-#: map/templates/map/mapview.html:40
+#: map/templates/map/mapview.html:18
 msgid "Our histories for our trip"
-msgstr "Ita nia istoria sira kona-ba ita nia viajen"
+msgstr "Ita nia istoria kona-ba ita nia viajen"
 
-#: map/templates/map/mapview.html:48
+#: map/templates/map/mapview.html:26
 msgid "Add my journey"
```
Comment (eraulo) [inline locale/tet/LC_MESSAGES/django.po:81]:
```suggestion
msgid "Add your journey"
```

File: locale/tet/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -40,69 +54,284 @@ msgstr "Tama"
...
+msgid ""
+"Forgotten your password? Enter your email address below, and we'll email "
+"instructions for setting a new one."
+msgstr ""
+"Haluha ita nia password? Hatama ita boot nia enderesu email iha kraik, no "
+"ami sei haruka instrusaun ba email ne'e hodi defini fali password foun."
+
+#: map/templates/registration/password_reset_form.html:16
+#| msgid "Change password"
+msgid "Reset my password"
```
Comment (eraulo) [inline locale/tet/LC_MESSAGES/django.po:311]:
```suggestion
msgid "Reset your password"
```

File: map/templates/registration/password_change_form.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                {{ form.new_password2.label_tag }}
+                {{ form.new_password2 }}
+                {{ form.new_password2.errors }}
+                {% if form.new_password2.help_text %}
+                <p class="help">{{ form.new_password2.help_text|safe }}</p>
+                {% endif %}
+            </div>
+        </fieldset>
+        <div class="form-row">
+            <button type="submit" class="btn btn-primary">{% trans 'Change my password' %}</button>
```
Comment (eraulo) [inline map/templates/registration/password_change_form.html:42]:
```suggestion
            <button type="submit" class="btn btn-primary">{% trans 'Change your password' %}</button>
```

**Pull request merged**